bentoml/_internal/server/runner_app.py

```diff
@@ -123,19 +123,19 @@ def client_response_hook(span: "Span", _message: t.Any) -> None:
             )
         )
 
-        from .access import AccessLogMiddleware
-
-        middlewares.append(
-            Middleware(
-                AccessLogMiddleware,
-                fields=[
-                    "REQUEST_CONTENT_TYPE",
-                    "REQUEST_CONTENT_LENGTH",
-                    "RESPONSE_CONTENT_TYPE",
-                    "RESPONSE_CONTENT_LENGTH",
-                ],
+        access_log_config = DeploymentContainer.runners_config.logging.access
+        if access_log_config.enabled.get():
+            from .access import AccessLogMiddleware
+
+            middlewares.append(
+                Middleware(
+                    AccessLogMiddleware,
+                    has_request_content_length=access_log_config.request_content_length.get(),
```

Comment:
nit: renaming `has_` to `log_`? e.g. from `has_request_content_length` to `log_request_content_length`